src/vector/index.js

```diff
@@ -136,9 +140,13 @@ window.onload = function() {
 
 function loadApp() {
     if (validBrowser) {
-        var MatrixChat = sdk.getComponent('pages.MatrixChat');
+        var MatrixChat = sdk.getComponent('structures.MatrixChat');
         window.matrixChat = ReactDOM.render(
-            <MatrixChat onNewScreen={onNewScreen} registrationUrl={makeRegistrationUrl()} />,
+            <MatrixChat
+                onNewScreen={onNewScreen}
+                registrationUrl={makeRegistrationUrl()}
+                ConferenceHandler={VectorConferenceHandler}
```

Comment:
I don't agree that just because we have `getComponent()` that we should be abusing it everywhere for things which are not React-style classes. I'm happy to move `VectorConferenceHandler` into react-sdk as we originally talked about with @ara4n if you wish.